### PR TITLE
JavaScript Conference Page

### PR DESCRIPTION
**Pull request by Piercel2022**
This pull request describe my capstone project. It is about building the International Java Script Conference website 2022 with the inspiration of the design guidelines of Cindy Shin in Behance. There are only two pages to build: The main index page and the "About" page in two views mobile and  desktop as shown below
mobile
desktop

### CODE REVIEW

**Review by Piercel2022 — COMMENTED** [review]
Update changes

File: README.md [hunk 1]
```diff
@@ -1 +1,80 @@
-# capstone1
+## Description of the project
```
Comment (zainsadaqat) [inline README.md:1]: ### Please add a Title for this project. 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @Piercel2022  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors

## Required Changes ♻️

- [ ] Please add a little more description to this Pull Request about what you have done in this branch 👍🏻 
- [ ] About page is missing. When I try to visit about page it says "Page Not Found"
Screenshot 2022-06-30 at 12 05 14 AM

- [ ] Menu items should be in the center. 👍🏻 
Screenshot 2022-06-30 at 12 07 02 AM

- [ ] There's a horizontal scroll bar in your project which makes it non responsive. Kindly fix this 👍🏻 
Screenshot 2022-06-30 at 12 08 23 AM

- [ ] Icon, Title, and Description is looking really congested. Please add some padding and make this section look consistent and responsive 👍🏻 
Screenshot 2022-06-30 at 12 09 17 AM

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rotimiazeez) [reply README.md:2]: - Kindly make sure this is done 👍🏽

**Review by mateo951 — APPROVED** [review]
### Hi @Piercel2022, 

Well done making all the changes requested by the previous reviewer 🔥
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it 🤓 

### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**